package.json

```diff
@@ -2,9 +2,13 @@
   "name": "preact-heroicons",
   "version": "2.0.13",
   "description": "Heroicons as Preact components",
-  "module": "dist/index.js",
-  "types": "dist/index.d.ts",
   "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
```

Comment:
The module keyword is not part of the package.json spec as mentioned here. 

https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for

exports is preferred. This was the most important change.